src/react-components/layout/Column.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import styles from "./Column.scss";
+
+export function Column({ as: Component, className, gap, padding, center, grow, overflow, children, ...rest }) {
+  const gapClass = gap === true ? styles.mdGap : styles[`${gap}Gap`];
+  const paddingClass = padding === true ? styles.lgPadding : styles[`${padding}Padding`];
```

Comment:
Very minor concern but it's a little surprising that the default values for `gap` and `padding` are different (`mdGap` and `lgPadding`). However I doubt this will ever cause us issues so if it feels good to write `gap` instead of `gap="md"` that's fine by me

Reply:
They just correspond to the standard spacing units I've come up with. I'll document them and move them to `theme.scss`

Reply:
Looks like things work really well with everything using the defaults of `md` and `lg`. Where `md` went from `20px` to `16px` and `lg` went from `24px` to `20px`. This actually fits the designs better too.